sections/03_categorize.md

```diff
@@ -44,92 +44,76 @@ regards to data availability and labeling.
-diagnosis based on a recent theoretical insight on the link between dropout
-networks and approximate Bayesian inference. Such developments are important for
-the whole medical image analysis field, because they have a potential to provide
-information about a level of confidence for every black-box algorithm's
-classification result and, thus, improve pathologist-computer interaction. Deep
-networks were also recently applied to age-related macular degeneration
-detection, similarly demonstrating the power of transfer learning when training
-set is limited [@tag:Burlina2016_amd] as well as the efficient use of a deep
-16-layer architecture combined with EMR data for training set enrichment.
-
-Mammography has been one area with numerous contributions
-[@tag:Dhungel2015_struct_pred_mamm @tag:Dhungel2016_mamm
-@tag:Zhu2016_mult_inst_mamm @tag:Zhu2016_advers_mamm
-@tag:Dhungel2017_mamm_min_interv]. In most of this work, researchers must work
-around a challenge typical for the domain - the limited number of well annotated
-training images. To expand the number and diversity of images, the researchers
-have employed approaches where they use adversarial examples
-[@tag:Zhu2016_advers_mamm] or first train towards human-created features before
-subsequent fine tuning [@tag:Dhungel2016_mamm]. Adaptation to the medical image
-domain can be further improved by combining in the latter approach with other
-machine learning techniques, for example, as a cascade of deep learning and
-random forest models [@tag:Dhungel2017_mamm_min_interv]. Using large dataset,
-Kooi et al. [@tag:Kooi2016_mamm_lesions] demonstrated that deep neural networks
-can outperform the traditional computer-aided diagnosis (CAD) system at low
-sensitivity and perform comparably at high sensitivity. They also compared
-network performance to certified screening radiologists on a patch level and
-found no significant difference between the network and the readers. Similarly,
-Geras et al. [@tag:Geras2017_multiview_mamm] showed that both using large
-dataset and using multi-view network architecture help to improve classification
-performance. The presence of a publicly available large bank of well-annotated
-mammography images would aid in the application of deep neural networks to this
-area and shift research focus from model generalization to effective processing
-of large image sets. Deep network pre-trained on large annotated mammogram set
-can be helpful for related tasks that do not have as much data using transfer
-learning [@tag:Kooi2017_mamm_tl]. Though this strategy has not yet been employed
-in this domain, high-quality feature detectors can be constructed from large
-collections of unlabeled images in an unsupervised context. The small number of
-labeled examples could be used for subsequent training. Similar strategies have
-been employed for EHR data where high-quality labeled examples are also
-difficult to obtain [@tag:BeaulieuJones2016_ehr_encode].
```

Comment:
The reference `tag:BeaulieuJones2016_ehr_encode` was removed from this section.  I think that's okay because the DOI `doi:10.1016/j.jbi.2016.10.007` still appears in other parts of Categorize.

I didn't detect any other references that were in the old version but not the new version so I reviewed the new text, ignoring the old.